include/dlaf/permutations/general/api.h

```diff
@@ -21,8 +22,9 @@ template <Backend B, Device D, class T, Coord coord>
 struct Permutations {
   static void call(SizeType i_begin, SizeType i_end, Matrix<const SizeType, D>& perms,
                    Matrix<T, D>& mat_in, Matrix<T, D>& mat_out);
-  static void call(comm::CommunicatorGrid grid, SizeType i_begin, SizeType i_end,
-                   Matrix<const SizeType, D>& perms, Matrix<T, D>& mat_in, Matrix<T, D>& mat_out);
+  static void call(common::Pipeline<comm::Communicator>& sub_task_chain, SizeType i_begin,
+                   SizeType i_end, Matrix<const SizeType, D>& perms, Matrix<T, D>& mat_in,
+                   Matrix<T, D>& mat_out);
```

Comment:
this signature does not seem to match...

Reply:
I don't understand the issue here, what doesn't match?

Reply:
Sorry, my fault. I was looking at the other side of the API, not the internal one 😉